### PR TITLE
Added server pagination tests

### PR DESCRIPTION
**Pull request by wilko55**
Gets us back to 100% coverage. Be good to get your thoughts 🤔

### CODE REVIEW

File: src/__tests__/components/Table.test.js [hunk 1]
```diff
@@ -234,4 +234,98 @@ describe('Table', () => {
...
+
+    describe('displayedRows', () => {
+        it('should equal row length when server pagination is used', () => {
+            jest.spyOn(tableActions, 'calculateRows').mockReturnValue([]);
+            const rows = [{ firstName: 'Paul', lastName: 'Darragh', isOpen: true }];
+            const pagination = {
+                isServerPagination: true,
+            }
+            wrapper = mount(<Table { ...props } />);
+            wrapper.setState({ pagination, rows });
```
Comment (JonForest) [inline src/__tests__/components/Table.test.js]: I think in this instance, and the test below, it makes sense to set `rows` and server registration via the `props` - as that is the normal mode of use.  Under non-test usage, a component will not have it's state set externally, so we should resist the temptation to do so here.

**Review by JonForest — CHANGES_REQUESTED** [review]
Minor change to two of the tests. 

Thanks,

File: src/__tests__/components/Table.test.js (hunk 1 above)
Comment (wilko55) [reply src/__tests__/components/Table.test.js]: Makes sense - have updated

**Review by JonForest — APPROVED** [review]
LGTM.
@wilko55 Feel free to merge, and thank you.

**Pull request merged**